requirements/edx/github.txt

```diff
@@ -90,6 +90,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
 git+https://github.com/edx/edx-user-state-client.git@1.0.1#egg=edx-user-state-client==1.0.1
 git+https://github.com/edx/edx-proctoring.git@0.12.14#egg=edx-proctoring==0.12.14
 git+https://github.com/edx/xblock-lti-consumer.git@v1.0.3#egg=xblock-lti-consumer==1.0.3
+git+https://github.com/edx/edx-django-extensions@c58817e95eb572f4ff57dad42728ac0b30f47dd2#egg=edx_management_commands
```

Comment:
Ugh. I will need to resolve some issues with edx-lint: https://github.com/edx/edx-lint/pull/38. In the interest of moving this along, I will leave edx/edx-django-extensions#2 open, and manually push this to PyPI.